home/service/search.py

```diff
@@ -31,15 +29,40 @@ def __init__(self, form: SearchForm, page: str, items_per_page: int = 20):
         self.paginator = self._get_paginator(items_per_page)
         self.context = self._get_context()
 
+    @staticmethod
+    def _build_filter_strings(
+        filter_param: str, filter_value_list: list[str]
+    ) -> list[str]:
+        return (
+            [f"{filter_param}{filter_value}" for filter_value in filter_value_list]
+            if filter_value_list
+            else []
```

Comment:
[Minor] I don't think the conditional is needed here - if `filter_value_list` is always a list then the list comprehension will return empty if it is empty

Reply:
Done